core/src/main/java/ch/cyberduck/core/http/DelayedHttpEntity.java

```diff
@@ -77,7 +77,7 @@ public DelayedHttpEntity(final Thread parentThread, final CountDownLatch streamO
     private final Thread parentThread;
 
     public boolean isRepeatable() {
-        return !entityWritten;
+        return false;
```

Comment:
Not sure if this affects other cases in a negative manner, i.e. https://github.com/iterate-ch/sardine/blob/master/src/main/java/com/github/sardine/impl/SardineImpl.java#L959

Reply:
Does not change semantics as the library will only query `isRepeatable` *after* invoking `writeTo` of the entity. Thus, the result is the same.

Reply:
Also in the mentioned case above which is about `Expect: 100-continue` with servers no supporting it. Entity is not written yet, I guess.

Reply:
Retry handled in 84e2f5e5.